conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/DropRuleInterpreter.scala

```diff
@@ -15,21 +15,27 @@
 
 package za.co.absa.enceladus.conformance.interpreter.rules
 
-import za.co.absa.enceladus.model.conformanceRule.DropConformanceRule
-import org.apache.spark.sql.Dataset
-import org.apache.spark.sql.Row
-import org.apache.spark.sql.SparkSession
-import za.co.absa.enceladus.dao.EnceladusDAO
+import org.apache.spark.sql.{Dataset, Row, SparkSession}
 import za.co.absa.enceladus.conformance.CmdConfig
-import za.co.absa.enceladus.utils.transformations.{ArrayTransformations, DeepArrayTransformations}
+import za.co.absa.enceladus.dao.EnceladusDAO
+import za.co.absa.enceladus.model.conformanceRule.DropConformanceRule
+import za.co.absa.enceladus.utils.transformations.DeepArrayTransformations
+
+import scala.util.{Failure, Success, Try}
 
 case class DropRuleInterpreter(rule: DropConformanceRule) extends RuleInterpreter {
 
   def conform(df: Dataset[Row])(implicit spark: SparkSession, dao: EnceladusDAO, progArgs: CmdConfig): Dataset[Row] = {
-    if (rule.outputColumn.contains('.')) {
-      conformNestedField(df)
-    } else {
-      conformRootField(df)
+    Try(df(rule.outputColumn)) match {
```

Comment:
It would probably better to inspect the schema.

See something like:
SchemaUtils.getFieldType(path: String, schema: StructType): Option[DataType]

Reply:
Did not know we had something for it. Like this?

Reply:
Yeah seems like a better approach to me

Reply:
I'm not sure I get what the benefit here is. @lokm01 ?

Reply:
Thanks, @lokm01 explained that it spares the overhead of throwing an exception and more importantly, catalyst optimizations can take place when checking if the field exists proactively.